hollow-explorer-ui/src/main/java/com/netflix/hollow/explorer/ui/HollowExplorerUI.java

```diff
@@ -105,13 +114,16 @@ public HollowReadStateEngine getStateEngine() {
             return client.getStateEngine();
         return stateEngine;
     }
-    
-    public void setHeaderDisplayString(String str) {
```

Comment:
is this going to break anyone else downstream?

Reply:
Yes. This will be a breaking change in Cinder-cli because we pass namespace to the explorer using this method.

Reply:
Another way to do it with minimal change is to just pass the namespace and env as space separated strings in headerDisplayString.